buildSrc/subprojects/build/build.gradle.kts

```diff
@@ -1,9 +1,10 @@
 dependencies {
+    api("com.google.guava:guava:26.0-jre")
+    api("org.asciidoctor:asciidoctor-gradle-plugin:1.5.9.2")
+    implementation("org.asciidoctor:asciidoctorj:1.5.8.1")
+    implementation("org.jruby:jruby-complete:9.2.5.0")
```

Comment:
This should be a constraint instead of a dependency (inside the `constraints {}` block)